node/resolvers/session/index.ts

```diff
@@ -102,6 +103,15 @@ export const mutations = {
       vtexIdCookies(ctx)
     )
 
+    const { items }: OrderForm = await checkout.orderForm()
+    const emptyItems = items.map(({ id, seller }: OrderFormItem, index) => ({
+      id,
+      seller,
+      quantity: 0,
+      index
+    }))
+    await checkout.updateItems(orderFormId, emptyItems)
```

Comment:
See my comment here: https://github.com/vtex-apps/store-discussion/issues/228#issuecomment-598872941

This will not be good solution, we would have to clear coupons, and other info that might get stuck in the orderForm.